model/src/w3sic1md.F90

```diff
@@ -1,5 +1,25 @@
+!> @file
+!> @brief Calculate ice source term S_{ice} according to simple methods.
+!>
+!> @author E. Rogers
+!> @author S. Zieger
+!> @date   11-Oct-2013
+!>
+
 #include "w3macros.h"
 !/ ------------------------------------------------------------------- /
+!>
+!> @brief Calculate ice source term S_{ice} according to simple methods.
+!>
+!> @author E. Rogers
+!> @author S. Zieger
+!> @date   11-Oct-2013
+!>
+!> @copyright Copyright 2009-2022 National Weather Service (NWS),
```

Comment:
Should the copyright be with the file? and not the module?

Reply:
Is there anything to distinguish this is the text for the module versus the file?  It looks like you duplicated text, but I'm assuming 1 block is for the file and the second is for the module.

Reply:
> Should the copyright be with the file? and not the module?

My recollection is we decided on including it with the module.  The original copyright statement does occur in the module header, so it makes sense from that.  For programs, I'm including it within the program section as well, vs. file.  I think we could do file if we like, and it would not be hard to make that adjustment even now.  I think it makes sense the way it is, but I could make the change no problem if that's what we decide.

Reply:
That sounds familiar now too!

Reply:
> Is there anything to distinguish this is the text for the module versus the file? It looks like you duplicated text, but I'm assuming 1 block is for the file and the second is for the module.

Yes, I am often duplicating text for 'file' and 'module'.  

I think it works out OK actually, because there is a 'module' view, and 'file' list view in the doxygen html output, and each has their 'brief's formatted for the respective pages.  So, it's actually the information you'd want if you found yourself in one section rather than the other (module or file list view).  I can say I didn't initially like the idea of the duplication you mentioned, but I actually came full circle on this.  You will probably find some files where the file brief says 'Contains MODULE xyz'.  This was what I came up with for files that had absolutely no text descriptions.  I then started using it even when there was text because I thought it added more (albeit little) than duplicating text from file.  Then when I started to review the html output, you have all these lines that say 'Contains MODULE ....', and I quickly found it annoying. This is when I realized that what seems like needless duplication is serving a purpose.  I've since switched from this, and in a future commit I plan to remove the 'Contains MODULE ---' lines, and replace them with the file brief text where that text exists.

Reply:
Thanks for the detailed explanation.